test/data/sandbox/ddt_test.ddt.js

```diff
@@ -1,18 +1,24 @@
+Feature('DDT');
```

Comment:
You can set datatable for feature by setting the same datatable for each Scenario inside a test.
This is much more usable:

```js
Data(accounts).Scenario('do smth with users')
Data(accounts).Scenario('do smth else')
```